x/gov/client/rest/query.go

```diff
@@ -379,10 +377,18 @@ func queryVotesOnProposalHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 
 		// For inactive proposals we must query the txs directly to get the votes
 		// as they're no longer in state.
+		params := types.NewQueryProposalVotesParams(proposalID, page, limit)
+
 		propStatus := proposal.Status
 		if !(propStatus == types.StatusVotingPeriod || propStatus == types.StatusDepositPeriod) {
 			res, err = gcutils.QueryVotesByTxQuery(cliCtx, params)
 		} else {
+			bz, err := cliCtx.Codec.MarshalJSON(params)
```

Comment:
```suggestion
			bz, err = cliCtx.Codec.MarshalJSON(params)
```

this should fix the linter warning

Reply:
Done!